intro to chapter 3.1 (#93)
thesis20/report: sections/experiments/methods/methods.tex

\subsection{State-of-the-art methods investigated}\label{subsec:sotaoverview}

> This section provides an in depth invistigation of the different methods that are examined in this paper.

Reviewer: ```suggestion
This section provides an overview of the different methods that will be examined in this paper.
```
